### ISSUE TITLE
Delete and Update records using RECORDNO instead of specific fields

### ISSUE THREAD

**Issue by MzDynamologic** [intacct/intacct-sdk-php#180]
When we create a record in Intacct, we get a RECORDNO for that entity. I am under the impression that this is the key field which can be used to keep track of that record.

Say I create a Customer or a Contact, I get a RECORDNO for the created record.

In the system that I am implementing, I would store this RECORDNO as intacct_id in the respective table (Customer, Contact) and use it to Update and Delete the record in Intacct as required.

However, when Updating or Deleting a Customer, we cannot update using the RECORDNO, we need to use CustomerId
Similarly, when Updating or Deleting a Contact, we use ContactName instead of the RECORDNO

Is there a way to Update and Delete records using the RECORDNO or is there any other field which exists for all entities that can be used for this purpose?

Looking at the Intacct documentation, Update and Delete operations are possible using the RECORDNO
https://developer.intacct.com/api/accounts-receivable/customers/#delete-customer
https://developer.intacct.com/api/company-console/contacts/#delete-contact

To summarize:

1) How can we Update a record using RECORDNO
2) How can we Delete a record using RECORDNO
3) How do we bulk delete multiple records using RECORDNO

I am using version 2.3.0 of the PHP SDK due to dependency constraints
